packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts

```diff
@@ -25,13 +25,31 @@ describe('Account', () => {
   const container = new MasterNodeRegTestContainer()
   const client = new ContainerAdapterClient(container)
   const createToken = createTokenForContainer(container)
+  const txtypes = [DfTxType.BURN_TOKEN, DfTxType.MINT_TOKEN]
+
+  let addr1: string
+  let addr2: string
 
   beforeAll(async () => {
     await container.start()
-    await container.waitForReady()
     await container.waitForWalletCoinbaseMaturity()
     await container.waitForWalletBalanceGTE(100)
-    await createToken(await container.getNewAddress(), 'DBTC', 200)
+
+    addr1 = await container.getNewAddress()
+    addr2 = await container.getNewAddress()
+
+    await createToken(addr1, 'dBTC', 201)
+    await createToken(addr2, 'dETH', 100)
+
+    for (let i = 1; i <= 10; i++) {
+      await client.token.burnTokens(`${0.01 * i}@dBTC`, addr1)
+      await container.generate(1)
+    }
+
+    for (let i = 1; i <= 10; i++) {
+      await client.token.burnTokens(`${0.01 * i}@dETH`, addr2)
+      await container.generate(1)
+    }
```

Comment:
Can it be done in a single loop?
```suggestion
    for (let i = 1; i <= 10; i++) {
      await client.token.burnTokens(`${0.01 * i}@dBTC`, addr1)
      await client.token.burnTokens(`${0.01 * i}@dETH`, addr2)
      await container.generate(1)
    }
```